components/scream/src/mct_coupling/scream_cxx_f90_interface.cpp

```diff
@@ -146,20 +138,46 @@ void scream_setup_surface_coupling (
     const auto& sc = ad.get_surface_coupling();
 
     // Register import/export fields
-    sc->set_num_fields(num_imports,num_exports);
+    sc->set_num_fields(num_imports,num_scream_imports,num_exports);
     for (int i=0; i<num_imports; ++i) {
       sc->register_import(names_in[i],x2a_indices[i]);
     }
     for (int i=0; i<num_exports; ++i) {
-      sc->register_import(names_out[i],a2x_indices[i]);
+      sc->register_export(names_out[i],a2x_indices[i]);
     }
 
     sc->registration_ends(cpl_x2a_ptr, cpl_a2x_ptr);
+  });
+}
+
+void scream_init_atm (const int& start_ymd,
+                      const int& start_tod)
+{
+  using namespace scream;
+  using namespace scream::control;
+
+  fpe_guard_wrapper([&](){
+    // First of all, initialize the scream session
+    scream::initialize_scream_session();
```

Comment:
I think this should be now moved inside scream_create_atm_instance, right?

Reply:
After this PR is finished, my plan was to have a separate one with all the stuff required for CIME to run. I can include those changes here if you'd rather keep them together.

Reply:
Maybe it makes sense to combine since this PR already includes interface changes..

Reply:
Up to you. If the SC work can be self-contained (it seems you are adding scream-standalone testing for it), then we can break it up into 2 prs..

Reply:
So, are you going to do it in this PR?

Reply:
I'll split it up.

Reply:
Fix in this PR: https://github.com/E3SM-Project/scream/pull/1152